compiler/src/iree/compiler/Codegen/Common/DecomposeConvolutionToLowerDimOps.cpp

```diff
@@ -16,6 +19,78 @@ namespace mlir::iree_compiler {
 
 namespace {
 
+/// Computes a "decomposed" lowering config attribute for a conv OP
+///
+/// This method complements the patterns to decompose 2D convolutions into 1D
+/// convs. Specifically, it will update the lowering config attached to a Conv
+/// Op in a way that matches the "decomposition" patterns.
+///
+/// At the moment only Depthwise HWC convolutions are supported.
+static FailureOr<IREE::Codegen::LoweringConfigAttr>
+computeNewLCA(SmallVector<Operation *> computeOps, MLIRContext *context) {
```

Comment:
Use an `ArrayRef` rather than copying the vector:

```suggestion
computeNewLCA(ArrayRef<Operation *> computeOps, MLIRContext *context) {
```